sphinx/cmd/quickstart.py

```diff
@@ -321,10 +323,11 @@ def ask_user(d: Dict[str, Any]) -> None:
     print()
 
 
-def generate(d: Dict, overwrite: bool = True, silent: bool = False, templatedir: str = None
-             ) -> None:
+def generate(
+    d: Dict, overwrite: bool = True, silent: bool = False, templatedir: Optional[str] = None
```

Comment:
i'd prefer to move the default empty str to the method signature here, but wasn't sure how you'd feel about changing the type signature

Reply:
I think no developers pass `None` value expectedly. So let's go ahead :-)

Reply:
it's called with `None` internally. If strict optionals were fully-implemented it would be easy to detect where this is the case and fix it, but right now it's a complex problem. Will revert this change. To be revisited in future